src/SprykerEco/Shared/AdyenApi/Transfer/adyen_api.transfer.xml

```diff
@@ -6,395 +6,395 @@
 
     <transfer name="AdyenApiRequest">
-        <property name="paymentMethodsRequest" type="AdyenApiGetPaymentMethodsRequest" />
-        <property name="makePaymentRequest" type="AdyenApiMakePaymentRequest" />
-        <property name="paymentDetailsRequest" type="AdyenApiPaymentDetailsRequest" />
-        <property name="authorizeRequest" type="AdyenApiAuthorizeRequest" />
-        <property name="authorize3dRequest" type="AdyenApiAuthorize3dRequest" />
-        <property name="captureRequest" type="AdyenApiCaptureRequest" />
-        <property name="cancelRequest" type="AdyenApiCancelRequest" />
-        <property name="refundRequest" type="AdyenApiRefundRequest" />
-        <property name="cancelOrRefundRequest" type="AdyenApiCancelOrRefundRequest" />
-        <property name="technicalCancelRequest" type="AdyenApiTechnicalCancelRequest" />
-        <property name="adjustAuthorizationRequest" type="AdyenApiAdjustAuthorizationRequest" />
+        <property name="paymentMethodsRequest" type="AdyenApiGetPaymentMethodsRequest"/>
+        <property name="makePaymentRequest" type="AdyenApiMakePaymentRequest"/>
+        <property name="paymentDetailsRequest" type="AdyenApiPaymentDetailsRequest"/>
+        <property name="authorizeRequest" type="AdyenApiAuthorizeRequest"/>
+        <property name="authorize3dRequest" type="AdyenApiAuthorize3dRequest"/>
+        <property name="captureRequest" type="AdyenApiCaptureRequest"/>
+        <property name="cancelRequest" type="AdyenApiCancelRequest"/>
+        <property name="refundRequest" type="AdyenApiRefundRequest"/>
+        <property name="cancelOrRefundRequest" type="AdyenApiCancelOrRefundRequest"/>
+        <property name="technicalCancelRequest" type="AdyenApiTechnicalCancelRequest"/>
+        <property name="adjustAuthorizationRequest" type="AdyenApiAdjustAuthorizationRequest"/>
     </transfer>
 
     <transfer name="AdyenApiGetPaymentMethodsRequest">
-        <property name="merchantAccount" type="string" />
-        <property name="countryCode" type="string" />
-        <property name="amount" type="AdyenApiAmount" />
-        <property name="channel" type="string" />
+        <property name="merchantAccount" type="string"/>
+        <property name="countryCode" type="string"/>
+        <property name="amount" type="AdyenApiAmount"/>
+        <property name="channel" type="string"/>
     </transfer>
 
     <transfer name="AdyenApiMakePaymentRequest">
         <!-- Required! -->
-        <property name="amount" type="AdyenApiAmount" />
-        <property name="merchantAccount" type="string" />
-        <property name="paymentMethod" type="array" />
-        <property name="reference" type="string" />  <!-- (transaction ID) max length 80 -->
-        <property name="returnUrl" type="string" />
+        <property name="amount" type="AdyenApiAmount"/>
+        <property name="merchantAccount" type="string"/>
+        <property name="paymentMethod" type="array" singular="paymentMethodItem"/>
```

Comment:
* This is a BC break.
* By default the generated method will have a name `addPaymentMethod()` and now it will be changed to `addPaymentMethodItem`, all code that could have been written on the client side will crash.

Reply:
Possible solutions are to roll back changes or `singular` has to be equal to `name` property in order to warranty the BC promise.